tests/integration/security/sds_ingress/util/util.go

```diff
@@ -644,3 +644,34 @@ func SetInstances(apps echo.Services) error {
 	}
 	return nil
 }
+
+// Some cloud platform may throw the following error during creation of the service with mixed TCP/UDP protocols:
+// "Error syncing load balancer: failed to ensure load balancer: mixed protocol is not supported for LoadBalancer".
+// Make sure the service is up and running before proceeding with the test.
+func WaitForIngressQUICService(t framework.TestContext, ns string) error {
+	return retry.UntilSuccess(func() error {
```

Comment:
Would it be easier to check if the backing pods are running rather than the service?

Reply:
@jaellio 

As I wrote in the comment above, since the service doesn't get any specific state and just stuck in a "Pending" state, it's pretty hard to know that the root cause of the "stuck" state.